docs/cli/geo/maps.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Maps
+description: Use Amplify CLI to create and manage maps to visualize GeoSpatial data in your app.
+---
+
+Amplify CLI's `geo` category enables you to create and manage map resources used to visualize GeoSpatial data in your application.
+You need to install Amplify CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
+```console
+npm i -g @aws-amplify/cli@geo
+```
+
+## Setup a new Map
+
+You can add a new map by running the following command from your project's root folder:
+
+```bash
+amplify add geo
+```
+```console
+? Select which capability you want to add:
+> Map (visualize the GeoSpatial data)
+  Location search (search by places, addresses, coordinates)
+```
+If you haven't set up the `auth` category already, the Amplify CLI will guide you through a workflow to enable the auth category.
+
+```console
+? You need to add auth (Amazon Cognito) to your project in order to add geo resources. Do you want to add auth now?
+> Yes
+
+Using service: Cognito, provided by: awscloudformation
+```
+
+Next, you can set a friendly name for the Map being provisioned. 
+```console
+? Provide a name for the Map:
+> StreetsMap
+```
+
+### Map Access permissions
+
+Next, configure the access permissions for your Map resource. You can scope permissions based on an individual user's authentication status.
+
+```console
+? Who can access this Map?
+❯ Authorized users only
+  Authorized and Guest users
+```
+
+Select `Authorized users only` if only authenticated users can have read-only permissions for the map.
```

Comment:
After reading the below, it doesn't sound like "read-only" is the right term. It's more like:
```suggestion
Select `Authorized users only` if only authenticated users are allowed to render the map.
```

Is my assumption correct?

Reply:
yeah these read-only permissions are used to render the map in the app